src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issueadjournment/IssueAdjournmentNoticeAboutToStartHandlerTest.java

```diff
@@ -89,10 +90,11 @@ public void givenAboutToStartRequest_willGeneratePreviewFile() {
     @Test
     public void givenNoPreviewDecisionFoundOnCase_thenShowError() {
         sscsCaseData.setAdjournCasePreviewDocument(null);
+        sscsCaseData.setAdjournCaseGenerateNotice("No");
         PreSubmitCallbackResponse<SscsCaseData> result = handler.handle(ABOUT_TO_START, callback, USER_AUTHORISATION);
 
         String error = result.getErrors().stream().findFirst().orElse("");
-        assertEquals("No draft adjournment notice found on case. Please use 'Adjourn case' event before trying to issue.", error);
+        assertEquals("No draft adjournment notice found on case. Please use 'Adjourn case' event or upload your adjourn case document.", error);
```

Comment:
As it is a short test file, would be really nice if you could replace the assertions with assertThat (junit Assertions). So here and the other tests using assertFalse.

Reply:
done